Update visdom handler

## 🐛 Bug description

[Visdom](https://github.com/fossasia/visdom) has updated its version from 0.1.8.9 to 0.2.1. The new version seems like failing some tests of the `VisdomHandler` here: https://github.com/pytorch/ignite/actions/runs/3201321175

```
=========================== short test summary info ============================
FAILED tests/ignite/contrib/handlers/test_visdom_logger.py::test_logger_init_env_vars
FAILED tests/ignite/contrib/handlers/test_visdom_logger.py::test_integration_no_executor
FAILED tests/ignite/contrib/handlers/test_visdom_logger.py::test_integration_with_executor
FAILED tests/ignite/contrib/handlers/test_visdom_logger.py::test_integration_with_executor_as_context_manager
ERROR tests/ignite/contrib/handlers/test_visdom_logger.py::test_logger_init_hostname_port
= 4 failed, 1474 passed, 417 skipped, 31 deselected, 11 warnings, 1 error in 339.13s (0:05:39) =
```

Need to upgrade the `VisdomHandler` to play with the new version of `visdom`.

--- a/tests/ignite/contrib/conftest.py
+++ b/tests/ignite/contrib/conftest.py
@@ -1,4 +1,5 @@
 import random
+from pathlib import Path
 
 import pytest
 
@@ -28,6 +29,7 @@ def visdom_server():
         from visdom import Visdom
         from visdom.server import download_scripts
 
+        (Path.home() / ".visdom").mkdir(exist_ok=True)
         download_scripts()
 
         vd_hostname = "localhost"

--- a/tests/ignite/contrib/handlers/test_visdom_logger.py
+++ b/tests/ignite/contrib/handlers/test_visdom_logger.py
@@ -857,7 +857,6 @@ def test_integration_no_server():
         VisdomLogger()
 
 
-@pytest.mark.skip(reason="temporary pass ci")
 @pytest.mark.skipif(sys.platform.startswith("win"), reason="Skip on Windows")
 def test_logger_init_hostname_port(visdom_server):
     # Explicit hostname, port
@@ -866,7 +865,6 @@ def test_logger_init_hostname_port(visdom_server):
     vd_logger.close()
 
 
-@pytest.mark.skip(reason="temporary pass ci")
 @pytest.mark.skipif(sys.platform.startswith("win"), reason="Skip on Windows")
 def test_logger_init_env_vars(visdom_server):
     # As env vars
@@ -885,7 +883,6 @@ def _parse_content(content):
     return json.loads(content)
 
 
-@pytest.mark.skip(reason="temporary pass ci")
 @pytest.mark.skipif(sys.platform.startswith("win"), reason="Skip on Windows")
 def test_integration_no_executor(visdom_server):
     vd_logger = VisdomLogger(server=visdom_server[0], port=visdom_server[1], num_workers=0)
@@ -922,7 +919,6 @@ def test_integration_no_executor(visdom_server):
     vd_logger.close()
 
 
-@pytest.mark.skip(reason="temporary pass ci")
 @pytest.mark.skipif(sys.platform.startswith("win"), reason="Skip on Windows")
 def test_integration_with_executor(visdom_server):
     vd_logger = VisdomLogger(server=visdom_server[0], port=visdom_server[1], num_workers=1)
@@ -960,7 +956,6 @@ def test_integration_with_executor(visdom_server):
     vd_logger.close()
 
 
-@pytest.mark.skip(reason="temporary pass ci")
 @pytest.mark.skipif(sys.platform.startswith("win"), reason="Skip on Windows")
 def test_integration_with_executor_as_context_manager(visdom_server, visdom_server_stop):
 

